.github/workflows/knative-releasability.yaml

```diff
@@ -40,31 +42,43 @@ jobs:
           go-version: 1.15.x
 
       - name: Install Dependencies
-        run: |
-          GO111MODULE=on go get tableflip.dev/buoy@master
+        run: GO111MODULE=on go get tableflip.dev/buoy@master
 
       - name: Check out code
         uses: actions/checkout@v2
+        with:
+          path: ./src/knative.dev/${{ github.event.repository.name }}
 
       - name: Check
+        run: buoy check go.mod --release ${RELEASE} --domain knative.dev --verbose
```

Comment:
We no longer notify when this fails 🙃 

I was thinking something along the lines of:
```shell
cat >> $GITHUB_ENV <<EOF
SLACK_MESSAGE(buoy check || CHECK=$?)
EOF
if [[ "$CHECK" -ne "0"  ]]; then
   echo 'notify=true' >> $GITHUB_ENV
fi

...

if [[ -z "$(git status --porcelain)" ]]; then
  ...
else
   echo 'notify=true' >> $GITHUB_ENV
   echo 'SLACK_MESSAGE=something about the dependencies being out of date'  >> $GITHUB_ENV
fi
```

Reply:
right arg. sorry